src/rpc/blockchain.cpp

```diff
@@ -1291,7 +1297,13 @@ static RPCHelpMan dumpcoinstats()
         file << strprintf("%s,", FormatMoney(unspendables_scripts));
 
         const CAmount unspendables_unclaimed_rewards = cumulative ? stats.total_unspendables_unclaimed_rewards : stats.total_unspendables_unclaimed_rewards - prev_stats.total_unspendables_unclaimed_rewards;
-        file << strprintf("%s\n", FormatMoney(unspendables_unclaimed_rewards));
+        file << strprintf("%s,", FormatMoney(unspendables_unclaimed_rewards));
+
+        file << strprintf("%d,", pindex->nTx);
```

Comment:
3772c09e16544c7876da13f33e6cb11401c9e4bd : this is a not cumulative, so would be nice to have cumulative version.

Reply:
done